modules/render.js

```diff
@@ -70,16 +81,26 @@ const render = () => {
 
     todoCheck.addEventListener('change', () => {
       if (todoCheck.checked) {
-        console.log('Checkbox is checked..', i);
-        updateStatus(i, true);
+        updateCheckBox(i, true);
+        todoDesc.style.textDecoration = 'line-through';
       } else {
-        console.log('Checkbox is not checked..');
-        updateStatus(i, false);
+        updateCheckBox(i, false);
       }
     });
   }
 };
 
+clearCompletedBtn.addEventListener('click', () => {
+  const todo = new Todo();
+  for (let i = 0; i < todosArr.length; i += 1) {
+    if (todosArr[i].completed) {
+      updateIndex(i);
+      todo.remove(i, todosArr);
+    }
+  }
+  render();
+});
```

Comment:
✅ A function for the "Clear all completed" button using the `Array.filter()` method is implemented 👍🏻.